src/main/java/edu/cornell/kfs/fp/businessobject/PurchasingDataBase.java

```diff
@@ -0,0 +1,525 @@
+    public KualiDecimal getUnitCost() {
+      return unitCost;
+    }
+
+
+    /**
+     * @param unitCost the unitCost to set
+     */
+    public void setUnitCost(KualiDecimal unitCost) {
+      this.unitCost = unitCost;
+    }
+
+
+    /**
+     * @param unitCost the unitCost to set
+     */
+    public void setUnitCost(String unitCost) {
+      this.unitCost = new KualiDecimal(unitCost);
+    }
+
+
+    /**
+     * @return the typeOfSupply
+     */
+    public String getTypeOfSupply() {
+      return typeOfSupply;
+    }
+
+
+    /**
+     * @param typeOfSupply the typeOfSupply to set
+     */
+    public void setTypeOfSupply(String typeOfSupply) {
+      this.typeOfSupply = typeOfSupply;
+    }
+
+
+    /**
+     * @return the serviceIdentifier
+     */
+    public String getServiceIdentifier() {
+      return serviceIdentifier;
+    }
+
+
+    /**
+     * @param serviceIdentifier the serviceIdentifier to set
+     */
+    public void setServiceIdentifier(String serviceIdentifier) {
+      this.serviceIdentifier = serviceIdentifier;
+    }
+
+
+    /**
+     * @return the messageIdentifier
+     */
+    public String getMessageIdentifier() {
+      return messageIdentifier;
+    }
+
+
+    /**
+     * @param messageIdentifier the messageIdentifier to set
+     */
+    public void setMessageIdentifier(String messageIdentifier) {
+      this.messageIdentifier = messageIdentifier;
+    }
+
+
+    /**
+     * @return the itemSequenceNumber
+     */
+    public Integer getItemSequenceNumber() {
+      return itemSequenceNumber;
+    }
+
+
+    /**
+     * @param itemSequenceNumber the itemSequenceNumber to set
+     */
+    public void setItemSequenceNumber(Integer itemSequenceNumber) {
+      this.itemSequenceNumber = itemSequenceNumber;
+    }
+
+
+    /**
+     * @param itemSequenceNumber the itemSequenceNumber to set
+     */
+    public void setItemSequenceNumber(String itemSequenceNumber) {
+      this.itemSequenceNumber = Integer.parseInt(itemSequenceNumber);
+    }
+
+
+    /**
+     * @return the lineItemDetailIndicator
+     */
+    public Integer getLineItemDetailIndicator() {
+      return lineItemDetailIndicator;
+    }
+
+
+    /**
+     * Documentation indicates that this must be one of two values:
+     * 1=Last line item detail record
+     * 2=Other line item detail record
+     * 
+     * However, review of the file indicates that we actually observe:
+     * 0=Last line item detail record
+     * 1=Other line item detail record
+     * 
+     * @param lineItemDetailIndicator the lineItemDetailIndicator to set
+     */
+    public void setLineItemDetailIndicator(Integer lineItemDetailIndicator) {
+      this.lineItemDetailIndicator = lineItemDetailIndicator;  
+    }
+
+
+    /**
+     * @param lineItemDetailIndicator the lineItemDetailIndicator to set
+     */
+    public void setLineItemDetailIndicator(String lineItemDetailIndicator) {
+      this.setLineItemDetailIndicator(Integer.parseInt(lineItemDetailIndicator));
```

Comment:
Another implicit int-to-Integer conversion, and also calls the Integer setter instead of setting the value directly like the other methods. Not sure if this is intentional.